debian/control

```diff
@@ -9,4 +9,5 @@ Standards-Version: 3.9.2
 Package: ros-humble-ros-workspace
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}, 
+Conflicts: libsystemd0 (<= 249.11-0ubuntu3), libudev1 (<= 249.11-0ubuntu3)
```

Comment:
Not a blocking comment, just curious if there is a reason to prefer `<= 249.11-0ubuntu3` over `< 249.11-0ubuntu3.1` which is what the systemd packages all list?

Reply:
Two reasons:
1. Any version, even a hypothetical one between `249.11-0ubuntu3` and ` 249.11-0ubuntu3.1` would be sufficient to satisfy this `Conflicts`. We don't specifically want the one currently in the updates repo, rather we specifically DON'T want the one in the original published repo. So I thought it best to call out that one specifically.
2. Debian calls for use of `<<` here instead of `<`, and I think that looks weird. It's silly.

Reply:
Either or both are fine by me.